scripts/extract-and-replace-translations.ts

```diff
@@ -0,0 +1,141 @@
+/**
```

Comment:
Niet om af te doen van deze oplossing want lijkt goed te werken 👍maar heb je ook gekeken naar i18next hun eigen oplossing hiervoor: https://github.com/i18next/i18next-scanner ?
Of waren er specifieke use cases die daar niet mee gaan?

Reply:
ja, die heb ik eerst geprobeerd

Maar daar kreeg je niets van context uit de ge-extracte strings. Die scanner exporteerde alles als:

toepassen: Toepassen
terug: Terug

maar daarmee weet je niet op welke pagina/view die string gebruikt wordt. Dus heb ik het pad toegevoegd aan de keys, zodat we beter kunnen zien waar die gebruikt worden.

Ook kan mijn oplossing stings extracten die voordien nog niet geextract zijn en die dan samen voegen met de reeds geextracte strings. :D